### PR TITLE
fix: chain: use latest go-state-types types for miner UI

### PR DESCRIPTION
**Pull request by arajasek**
## Related Issues

@jimpick pointed out that the SectorOnChainInfo returned over the API has the wrong field names.

## Proposed Changes

Use {{.latestVersion}} types which will keep us up to date.

## Checklist

Before you mark the PR ready for review, please make sure that:

- [ ] Commits have a clear commit message.
- [ ] PR title is in the form of of `<PR type>: <area>: <change being made>`
  - example: ` fix: mempool: Introduce a cache for valid signatures`
  - `PR type`: fix, feat, build, chore, ci, docs, perf, refactor, revert, style, test
  - `area`, e.g. api, chain, state, market, mempool, multisig, networking, paych, proving, sealing, wallet, deps
- [ ] If the PR affects users (e.g., new feature, bug fix, system requirements change), update the CHANGELOG.md and add details to the UNRELEASED section.
- [ ] New features have usage guidelines and / or documentation updates in
  - [ ] [Lotus Documentation](https://lotus.filecoin.io)
  - [ ] [Discussion Tutorials](https://github.com/filecoin-project/lotus/discussions/categories/tutorials)
- [ ] Tests exist for new functionality or change in behavior
- [ ] CI is green

### CODE REVIEW

File: documentation/en/api-v0-methods.md [hunk 1]
```diff
@@ -5875,12 +5875,12 @@ Response:
     "InitialPledge": "0",
     "ExpectedDayReward": "0",
     "ExpectedStoragePledge": "0",
-    "ReplacedSectorAge": 10101,
+    "PowerBaseEpoch": 10101,
```
Comment (Stebalien) [inline documentation/en/api-v0-methods.md:5878]: Is this not going to be an issue for API stability? And/or historical state?

**Review by Stebalien — APPROVED** [review]

File: documentation/en/api-v0-methods.md (hunk 1 above)
Comment (arajasek) [reply documentation/en/api-v0-methods.md:5878]: I'm honestly not sure? But it _is_ a breaking change that we've had in the protocol itself, so I feel like the API needs to break too here.

Comment (Stebalien) [reply documentation/en/api-v0-methods.md:5878]: Yeah, it does need to change. I guess: do it and see if anyone cares?

**Review by Stebalien — APPROVED** [review]

**Comment by jimpick** [comment]
The `PowerBaseEpoch` and `ReplacedDayReward` fields weren't being populated, so here's a PR for that:

https://github.com/filecoin-project/lotus/pull/11586

**Pull request merged**